deployments/local/shared/graphana/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM grafana/grafana
-
+USER root
 RUN apt-get update && apt-get -y install curl
```

Comment:
Without `USER root`, the `apt-get` commands would fail with "permission denied". Just me? Maybe a new version of the `graphana` image?